DigitalLearningSolutions.Web/Views/Shared/SearchablePage/_SearchResultsCount.cshtml

```diff
@@ -1,14 +1,11 @@
-﻿@using DigitalLearningSolutions.Web.ViewModels.Frameworks
-@model BaseFrameworksPageViewModel
-
-<p class="nhsuk-u-margin-top-4 nhsuk-u-margin-bottom-0">
-  @if (Model.SearchString == null)
-  {
-    <span hidden aria-hidden="true" aria-live="polite" id="results-count">0 matching results</span>
-  }
-  else
-  {
-    var count = Model.MatchingSearchResults;
-    <span role="alert" id="results-count">@count matching @(count == 1 ? "result" : "results")</span>
-  }
-</p>
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@model BaseSearchablePageViewModel
+
+<p class="nhsuk-u-margin-top-4 nhsuk-u-margin-bottom-0">
+  @if (Model.SearchString == null) {
+    <span hidden aria-hidden="true" aria-live="polite" id="results-count">0 matching results</span>

```

Comment:
I am assuming that Model.SearchString is only null when nothing has been searched for - if this is the case, do we want this hidden span? Is it necessary for js?

Reply:
I would assume so, but I'm not sure. This isn't new, it's refactored from the deleted /LearningPortal/Shared/_SearchResultCount.cshtml and /Frameworks/Shared/_SearchResultCount.cshtml without changing the content